src/components/SlateEditor/plugins/h5p/SlateH5p.tsx

```diff
@@ -143,14 +91,24 @@ const SlateH5p = ({ element, editor, attributes, language, children }: Props) =>
             <FigureButtons>
               {embed.status === 'success' && (
                 <ModalTrigger>
-                  <IconButtonV2
-                    colorTheme="light"
-                    onClick={() => setIsEditing(true)}
-                    title={t('form.editH5p')}
-                    aria-label={t('form.editH5p')}
-                  >
-                    <Link />
-                  </IconButtonV2>
+                  <>
```

Comment:
`ModalTrigger` på linjen over vil legge props på barnet som sendes til den. I dette tilfellet vil den legge props på fragment, som ikke er så veldig heldig. En modal skal se ca sånn her ut:

```
<Modal>
  <ModalTrigger>
    <Button>Open me</Button>
  </ModalTrigger>
  <ModalContent>Ting i modal!</ModalContent>
</Modal>
```

Det er ganske restriktivt mønster, men det sørger for at fokus-håndtering fungerer som det skal.

Reply:
Ait da drar jeg de ut i egne filer og bruker separate modaler :+1: